main/src/main/java/org/mobilitydata/gtfsvalidator/cli/Main.java

```diff
@@ -19,19 +19,21 @@
 import com.beust.jcommander.JCommander;
 import com.google.common.base.Strings;
 import com.google.common.flogger.FluentLogger;
+import org.mobilitydata.gtfsvalidator.input.GtfsFeedName;
```

Comment:
Please revert this formatting change and setup IDEA to use https://github.com/google/google-java-format (see instructions there).